### DISCUSSION TITLE
Is there any way to distinguish between changed/modified and added?

### DISCUSSION THREAD

**Discussion by jstuessy** [tj-actions/changed-files#2283] [Q&A] [answered]
I'm trying to ensure a step only runs is a file is changed in a directory, but not to run when new files are added.

**Comment by jackton1** [comment, marked as answer]
Yes each one has a separate output, for example to get a list of only modified files that already exists. See: https://github.com/tj-actions/changed-files?tab=readme-ov-file#output_modified_files

**Reply by jstuessy** [reply]
I'm so blind, thank you!
